.github/workflows/github-actions-demo.yml

```diff
@@ -0,0 +1,24 @@
+name: GitHub Actions Demo
+on: [push, pull_request]
+jobs:
+  Explore-GitHub-Actions:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
```

Comment:
I happen to know there's a v3 for this release action, perhaps use that instead? When implementing something new I think it's worth trying to use the most recent versions :)

Reply:
:+1: